chain/indexer/src/lib.rs

```diff
@@ -43,7 +45,7 @@ pub struct InitConfigArgs {
     /// Specify a custom download URL for the records file.
     pub download_records_url: Option<String>,
     /// Download the verified NEAR config file automatically.
-    pub download_config: bool,
+    pub download_config: Option<String>,
```

Comment:
nit: why not just make this field an `Option<DownloadConfigType>`? The only place this is used is in the one place in the indexer example, where we are allocating a string and writing "rpc" to it, and then calling `DownloadConfigType::from_str()` on the string we just allocated. we already know what's gonna be in the string bc we just wrote to it. you could just change indexer/example/src/configs.rs to say this instead:

```rust
if config_args.download_config {
  Some(DownloadConfigType::Rpc)
} else {
  None
}
```

Or you could even leave it as-is as a bool, and in `indexer_init_configs()` check that unchanged bool field and set it to `DownloadConfigType::Rpc` if it's true.

Reply:
I did not do it due to a dependency change, but I found a good place (near-config-utils) to put the new struct, so done with the suggestion, making the type `Option<DownloadConfigType>`.